DESCRIPTION

```diff
@@ -27,16 +27,17 @@ Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.3.1
 Imports: 
     dplyr,
+    jsonvalidate,
     logger,
-    pacta.portfolio.allocate,
+    pacta.portfolio.allocate (>= 0.0.2.9001),
     pacta.portfolio.audit,
     pacta.portfolio.import,
     pacta.portfolio.utils,
     pacta.workflow.utils,
     readr
 Remotes:
-    RMI-PACTA/pacta.portfolio.allocate,
+    RMI-PACTA/pacta.portfolio.allocate@fix/48-remove-globalenv,
     RMI-PACTA/pacta.portfolio.audit,
     RMI-PACTA/pacta.portfolio.import,
     RMI-PACTA/pacta.portfolio.utils,
-    RMI-PACTA/pacta.workflow.utils
+    RMI-PACTA/pacta.workflow.utils@fix/integration-issues-with-worfklow.pacta
```

Comment:
do you want the branch names merged to main?

Reply:
I'd rather not. Those are the two PRs mentioned at the top.

Reply:
resolved.